conda_pack/scripts/windows/activate.bat

```diff
@@ -0,0 +1,51 @@
+@REM @ symbols in this file indicate that output should not be printed.
+@REM   Setting it this way allows us to not touch the user's echo setting.
+@REM   For debugging, remove the @ on the section you need to study.
+@setlocal enabledelayedexpansion
+
+@CALL :NORMALIZEPATH "%~dp0"
+@SET "SCRIPT_DIR=%RETVAL%"
+@CALL :NORMALIZEPATH "%SCRIPT_DIR%\.."
+@set "NEW_PREFIX=%RETVAL%"
+
+@if "%CONDA_PREFIX%" == "" @goto skipdeactivate
+@if "%CONDA_PREFIX%" == "%NEW_PREFIX%" @exit /b
+
+@REM If the current environment is a conda-pack environment, or a root environment
+@REM
+@if exist "%CONDA_PREFIX%\Scripts\deactivate.bat" @call "%CONDA_PREFIX%\Scripts\deactivate.bat"
+@REM Newer versions of conda only have the deactivate script in the root environment
+@if exist "%CONDA_PREFIX%\..\..\Scripts\deactivate.bat" @call "%CONDA_PREFIX%\..\..\Scripts\deactivate.bat"

```

Comment:
I tried doing just `@CALL deactivate`/`@CALL deactivate.bat` here, but that doesn't seem to work.